thunderfish/chirp.py

```diff
@@ -0,0 +1,196 @@
+"""
+Detection of chirps in weakly electric fish recordings.
+
+chirp_analysis(): calculates spectrogram, detects fishes and extracts chirp times(combined for all fishes).
+                  !!! recommended for short recordings (up to 5 min) where only the chirp times shall be extracted !!!
+chirp_detection(): extracts chirp times with help of given spectrogram and fishlist.
+chirp_spectrogram(): performs spectrogram suitable for chip detection.
+"""
+
+import sys
+import numpy as np
+import matplotlib.mlab as mlab
+import dataloader as dl
+import harmonicgroups as hg
+import config_tools as ct
+import powerspectrum as ps
+import matplotlib.pyplot as plt
+import peakdetection as pkd
```

Comment:
most of those imports are only needed for the main part of the module. Move them there.